src/icons/IconPlusCircle.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export function IconPlusCircle() {
```

Comment:
@nicolasshiken make this to accept a size prop, this will be replaced later with proper impl